bin/style-dictionary

```diff
@@ -64,12 +65,19 @@ function styleDictionaryBuild(options) {
   // Create a style dictionary object with the config
   var styleDictionary = StyleDictionary.extend( configPath );
 
-  if (options.platform && options.platform.length > 0) {
-    options.platform.forEach(function(platform) {
-      styleDictionary.buildPlatform( platform );
-    });
-  } else {
-    styleDictionary.buildAllPlatforms();
+  try {
+    if (options.platform && options.platform.length > 0) {
+      options.platform.forEach(function(platform) {
+        styleDictionary.buildPlatform( platform );
+      });
+    } else {
+      styleDictionary.buildAllPlatforms();
+    }
+    ErrorHandler.flush();
+  }
+  catch (e) {
+    ErrorHandler.flush();
```

Comment:
Can all this stuff and the below in this file be removed? Should the style dictionary node module already flush out all the errors? If not it probably should so using the node module and the CLI should behave the same.